### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by drupalsunil**

### CODE REVIEW

**Review by drupalsunil — COMMENTED** [review]
ok

File: app/Http/Controllers/Admin/OrderController.php [hunk 1]
```diff
@@ -49,115 +49,135 @@ public function prod_name(Request $request){
 
             }
 
-            return response()->json($output) ;
-
+            // $output = [];
```
Comment (drupalsunil) [inline app/Http/Controllers/Admin/OrderController.php:52]: remove the commented code

File: app/Http/Controllers/Admin/OrderController.php [hunk 2]
```diff
@@ -16,30 +16,30 @@
...
+            $data = Product::with('category', 'ProductVeriant')
+                ->where('Title', 'LIKE', '%' . $search . '%')->where('Stock', '>=', 1)->orWhere('Generic_name', 'LIKE', '%' . $search . '%')->orderBy('title', 'asc')->orderBy('Exp_date','asc')->take(10)->get();
 
 
             $output =[];
             if (count($data)>0) {
                 //dump($data);
                 foreach($data as $d) {
                     $output[]= ['label'=>$d->Title,'id' =>$d->id,'values'=>$d];
-
```
Comment (drupalsunil) [inline app/Http/Controllers/Admin/OrderController.php:42]: remove white space

File: app/Models/Med_Function.php [hunk 3]
```diff
@@ -10,5 +10,6 @@ class Med_Function extends Model
     use HasFactory;
     protected $table = 'med__functions';
     protected $primaryKey = 'id';
-    
+    // public $fillable = ['Name'];
```
Comment (drupalsunil) [inline app/Models/Med_Function.php:13]: removed the use less code

**Review by drupalsunil — COMMENTED** [review]
review the comments

**Review by drupalsunil — COMMENTED** [review]
review the comments

**Review by rakolsoft — APPROVED** [review]

**Pull request merged**